### PR TITLE
Persian language

### PR DESCRIPTION
**Pull request by CodeWithEmad**
This will add Persian (fa_IR) language to the Ora Grading MFE.
Additionally, a few typos have been corrected.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @CodeWithEmad!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by e0d** [comment]
@CodeWithEmad looks like there are some test failures, can you have a look?

**Comment by CodeWithEmad** [comment]
sure @e0d. Fixed and tested on my local.
Can you please rerun the tests?

**Comment by mphilbrick211** [comment]
Hi @openedx/content-aurora! Would someone be able to please review/merge this for us? Thanks!

**Review by OmarIthawi — CHANGES_REQUESTED** [review]
Thanks @CodeWithEmad again and appologies for the late review requesting changes.

Same here please. The commit `chore: Persian translations added` which adds the translations `.json` file should be removed.

I've only skimmed through the typos, but overall the pull request looks good.

**Comment by CodeWithEmad** [comment]
@OmarIthawi I removed other language translations.
As I mentioned in our Slack chat, If you want to introduce a new language to any of the repos, you have to add the `LANGUAGE_CODE.json` file. otherwise how you can import it into the `src/i18n/index.js`? tests (specifically lint) will fail if you do so. 

` 8:26  error  Unable to resolve path to module './messages/fa_IR.json'  import/no-unresolved`

**Review by leangseu-edx — APPROVED** [review]
LGTM 👍

**Comment by CodeWithEmad** [comment]
@leangseu-edx your last commit here has conflicts with the master branch.

File: src/i18n/messages/fa_IR.json [hunk 1]
```diff
@@ -0,0 +1,105 @@
+{
```
Comment (OmarIthawi) [inline src/i18n/messages/fa_IR.json]: I recommend adding a empty file here. This file will be overridden by the translations bot anyway.

Comment (CodeWithEmad) [reply src/i18n/messages/fa_IR.json]: :) sure

Comment (CodeWithEmad) [reply src/i18n/messages/fa_IR.json]: Persian translations were removed. should I remove the conflicted commit @leangseu-edx added?

Comment (OmarIthawi) [reply src/i18n/messages/fa_IR.json]: I'm not sure about the `package.json` files. But generally, the preference is to make it in a separate pull request to make your contributions atomic.

Comment (CodeWithEmad) [reply src/i18n/messages/fa_IR.json]: Agree, @leangseu-edx add that commit. I'll remove package.json and package-lock.json from his commit and this should be ready to review.

**Review by OmarIthawi — APPROVED** [review]
Thanks @CodeWithEmad. i18n changes looks great! Thanks again for your contribution.

File: src/i18n/messages/fa_IR.json (hunk 1 above)
Comment (leangseu-edx) [reply src/i18n/messages/fa_IR.json]: sorry, for late response. The reason I update the package was that this pr wasn't compile locally.

**Comment by openedx-webhooks** [comment]
@CodeWithEmad 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-ora-grading&entry.867055334=https://github.com/openedx/frontend-app-ora-grading/pull/246&entry.1484655318=https://github.com/CodeWithEmad&entry.752974735=2023-08-08+16:28&entry.1917517419=2023-09-11+13:15&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**